guide/16-geoenrichment-guides/part1_introduction_to_geoenrichment.ipynb

```diff
@@ -0,0 +1,972 @@
+{
```

Comment:
<p>"In this part of the Geoenrichment guide series, you (have seen) <s>saw </s>a high-level example of how <code>arcgis.geoenrichment</code> module can be used to <code>enrich</code> a dataset with various socio-demographic features(, and also an introduction of)<s>. This part also introduces</s> different ways in which data can be enriched"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>this is fixed.</p>